packages/core/src/provisioning.ts

```diff
@@ -351,4 +378,154 @@ export class BasicProvisioner implements Provisioner {
 
         return prov;
     }
+
+    getAccountIdFromScimRequest(httpReq: IncomingMessage) {
+        const url = new URL(`http://localhost${httpReq.url || ""}`);
+        const secretToken = url.searchParams.get("token") || "";
+
+        // TODO: find secret token in the DB
+        if (secretToken === "asdrtyghj") {
+            return {
+                accountId: "472478c5-17e8-4ed5-8f51-05a9c5deaebb",
+                orgId: "e0bb91b4-2b35-4ba7-ba60-f4ac8470e7a3",
+            };
+        }
```

Comment:
This is just some local test data for me, but this is what I'd envision we return from a token.